docs/en/operations/table_engines/versionedcollapsingmergetree.md

```diff
@@ -0,0 +1,229 @@
+<a name="table_engine-versionedcollapsingmergetree"></a>
+
+# VersionedCollapsingMergeTree
+
+The engine:
+
+- Allows quick writing of continually changing states of objects.
```

Comment:
It'd be useful to briefly mention Version right here as a mechanism to restore order of writes coming from concurrent writers.